.github/workflows/test.yaml

```diff
@@ -88,7 +115,8 @@ jobs:
   ### real jobs after this
 
   process-managers:
-    needs: start-time
+    needs: [start-time, detect_jobs_to_run]
+    if: ${{ fromJson(needs.detect_jobs_to_run.outputs.jobs)['process-managers'] == true
```

Comment:
`process-managers` should be available as an ENV var here, making this copy pastable to all jobs without adaptations (which we would mess up sooner or later)

I _think_ it is `${{ github.job }}` (also used further down in "Set test specific DATABASE_URL" step)

Reply:
Good point!

Reply:
It didn't work :meh:

Reply:
Why not? The syntax to use it in there is probably different, maybe just `if: ${{ fromJson(needs.detect_jobs_to_run.outputs.jobs)[github.job] == true` - was that one of the variants you tried? (I never get these variable substitutions right on the first try)

Reply:
I tried this and it didn't work [`385377c` (#2286)](https://github.com/prisma/e2e-tests/pull/2286/commits/385377cb114d0b953c7f77a56ab433704fa098f6)
```
if: ${{ fromJson(needs.detect_jobs_to_run.outputs.jobs)['process-managers'] == true }}
```